src/main/java/org/fcrepo/migration/Migrator.java

```diff
@@ -18,8 +19,8 @@
  */
 public class Migrator {
 
-    public static void main(String [] args) throws IOException, XMLStreamException {
-        if (args.length == 1) {
+    public static void main(final String [] args) throws IOException, XMLStreamException {
```

Comment:
Thanks for this clean-up throughout the code!  I should probably start enforcing fcrepo4 code style rules sooner rather than later...